registry/registry.go

```diff
@@ -219,6 +220,10 @@ func ContinueOnError(err error) bool {
 		return shouldV2Fallback(v)
 	case *client.UnexpectedHTTPResponseError:
 		return true
+	case error:
+		if val := strings.Contains(err.Error(), errNoSpaceOnDevice.Error()); val {
```

Comment:
@cpuguy83, The error returned is from `chrootarchive.invokeUnpack()` function, which returns a error string `"Untar re-exec error: exit status 1: output: write /lib/udev/hwdb.d/20-pci-vendor-model.hwdb: no space left on device"` instead of `syscall.ENOSPC`. Is there a better way to obtain the error `ENOSPC` in the caller function ?